api/conf/conf.yaml

```diff
@@ -80,6 +80,17 @@ authentication:
     - username: user
       password: user
 
+oidc:
+  secret:
+    secret
+  expire_time: 3600
+  apps:
+    - app_name: authing
```

Comment:
I think we can support one app first.
So just remove apps.

Reply:
ok